RiddleBot/src/main/java/io/kemper/AnswerHandler.java

```diff
@@ -0,0 +1,56 @@
+package io.kemper;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.LambdaLogger;
+import com.amazonaws.services.lambda.runtime.RequestStreamHandler;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.kemper.api.Response;
+import io.kemper.db.DBHandler;
+import io.kemper.db.SimpleDBHandler;
+
+import java.io.*;
+import java.util.Map;
+
+
+public class AnswerHandler implements RequestStreamHandler {
+    DBHandler dbHandler;
+
+    public AnswerHandler() {
+        this.dbHandler = new SimpleDBHandler();
+    }
+
+    @Override
+    public void handleRequest(InputStream input, OutputStream output, Context context) throws IOException {
+        BufferedReader reader = new BufferedReader(new InputStreamReader(input));
+        OutputStreamWriter writer = new OutputStreamWriter(output, "UTF-8");
+        ObjectMapper mapper = new ObjectMapper();
+        JsonNode event = mapper.readTree(reader);
```

Comment:
**Consider alternative:** create a POJO to represent the JSON

See: [example](https://github.com/KemperIO/slack-riddle/blob/master/RiddleBot/src/main/java/io/kemper/api/Response.java)